scripts/cibuild/ConfigureVstsBuild.ps1

```diff
@@ -98,17 +110,25 @@ $NuGetLocalization = Join-Path $Submodules NuGet.Build.Localization -Resolve
 
 # Check if there is a localization branch associated with this branch repo
 $currentNuGetBranch = $env:BUILD_SOURCEBRANCHNAME
-$lsRemoteOpts = 'ls-remote', 'origin', $currentNuGetBranch
-Write-Host "Looking for branch '$currentNuGetBranch' in NuGet.Build.Localization"
-$lsResult = & git -C $NuGetLocalization $lsRemoteOpts
-
-if ($lsResult)
+if (Get-LocBranchExists $currentNuGetBranch)
 {
     $NuGetLocalizationRepoBranch = $currentNuGetBranch
 }
 else
 {
-    $NuGetLocalizationRepoBranch = 'dev'
+    if ($currentNuGetBranch -like "*-MSRC") {
+        $currentNuGetBranch = $currentNuGetBranch -replace "-MSRC$", ""
+        if (Get-LocBranchExists $currentNuGetBranch) {
+            $NuGetLocalizationRepoBranch = $currentNuGetBranch
+        }
+        else
+        {
+            $NuGetLocalizationRepoBranch = "dev"
```

Comment:
Console output something? This is an odd situation, right? Maybe we should just fail the script?

Reply:
Line 133 outputs the selected branch